python/lsst/ts/watcher/filtered_topic_wrapper.py

```diff
@@ -118,29 +122,49 @@ def __init__(self, model, topic, filter_field):
         # Data cache: a dict of filter_value: data
         self.data_cache = dict()
 
-        # dict of filter_value: list of field wrappers
+        # Field wrapper cache:
+        # a dict of filter_value: list of FilteredFieldWrapper
         self.field_wrappers = collections.defaultdict(list)
 
+        self.call_event = asyncio.Event()
+
         self.topic.callback.add_topic_wrapper(self)
 
         model.filtered_topic_wrappers[key] = self
 
     def add_field_wrapper(self, field_wrapper):
-        """Add a field wrapper to the internal cache.
+        """Add a filtered field wrapper to the internal cache.
 
-        Call field_wrapper.update_value when update_data is called
-        with the appropriate filter_value.
+        Parameter
+        ---------
+        field_wrapper : `BaseFilteredFieldWrapper`
+            The filtered field wrapper to add.
         """
         self.field_wrappers[field_wrapper.filter_value].append(field_wrapper)
 
     def get_data(self, filter_value):
         """Get the most recently seen data for the given filter_value,
         or None if no data seen.
+
+        Parameters
+        ----------
+        filter_value : any
```

Comment:
A friendly suggestion to enclose the python type in back ticks here.

Reply:
In this case "any" can't make a useful link. I replaced "any" with "scalar", as being more accurate (but still not linkable).